src/Container/PackageProxyContainer.php

```diff
@@ -32,7 +33,7 @@ public function __construct(Package $package)
      * @param string $id
      * @return mixed
      *
-     * @throws \Exception
+     * @throws ContainerException | NotFoundException
```

Comment:
It violates the Liskov substitution principle because `$this->container` is typed as  `\Psr\Container\ContainerInterface`. This means the `get` method could throw any `\Psr\Container\ContainerExceptionInterface`.

Reply:
Thanks for this comment @shvlv ! Perhaps we could change the type hint here to `\Psr\Container\ContainerExceptionInterface`

Reply:
```suggestion
     * @throws ContainerException|NotFoundException
```

For readability!